src/coreclr/vm/reflectioninvocation.cpp

```diff
@@ -1024,7 +813,7 @@ FCIMPL5(Object*, RuntimeMethodHandle::InvokeMethod,
             argDest = ArgDestination(pStackCopy, 0, NULL);
         }
 
-        InvokeUtil::CopyArg(th, &objs->GetAt(i), &argDest);
+        InvokeUtil::CopyArg(th, objs[i], &argDest);
```

Comment:
I assume that this will move to the managed size when we are preparing the ByRefs as well?

Reply:
Yes as part of the byref normalizing. Thanks